.github/ISSUE_TEMPLATE/documentation_template.md

```diff
@@ -0,0 +1,40 @@
+# Documentation 
+
+*title or suggestion of documentation*
+
+## Purpose
+
+- [ ] update
+- [ ] new 
+- [ ] remove
+- [ ] typo/grammar change
+- [ ] markdown improvement
+
+## Category 
+
+- [ ] code documentation
+- [ ] wiki
+- [ ] README
+- [ ] installation guide
+    - [ ] OS specifics
+    - [ ] dependency
+- [ ] Data products information 
+- [ ] tutorials
+    - [ ] plotting
+    - [ ] io
+    - [ ] data products
+- [ ] examples
+- [ ] citing/licensing
+- [ ] trouble shooting
+    - [ ] installation
+    - [ ] tutorials
+- [ ] history
+- [ ] guidelines
+    - [ ] developer
+    - [ ] communication
+    - [ ] release
```

Comment:
Just a note that I suggested removing these lines in the PR template for documentation, but perhaps they can stay in the issue template so that there's a way to report issues with guidelines. I'm not sure - it's harmless to include it.

Reply:
I think I kept them in both places just because we don't always have issues with PRs and vice versa. Like I didn't make a GRID issue to update the documentation :p 

My hope is if the issue is made then in the PR they reference it and maybe delete this section.

Reply:
I think issues can raise things that may not always result in a PR, so it's this is good as is, and I'm going to resolve the issue.